src/files.js

```diff
@@ -1205,5 +1205,7 @@ FOAM_FILES([
   { name: "foam/nanos/ndiff/NDiffJournal", flags: [ "java" ] },
   { name: "foam/nanos/ndiff/NDiffRuntimeDAO", flags: [ "java" ] },
   { name: "foam/u2/view/ComparisonView" },
-  { name: "foam/u2/view/SuggestedTextField", flags: [ "web" ] }
+  { name: "foam/u2/view/SuggestedTextField", flags: [ "web" ] },
+  { name: "foam/demos/csv/CsvUploadDemo", flags: ["web"] }
```

Comment:
Are we adding demo files to files.js as well?

Reply:
Yes, please add "demo" flag to flags: for demos. Then we should not include these in production builds.